__docs__/wonder-blocks-tokens/tokens-media-queries.mdx

```diff
@@ -0,0 +1,82 @@
+import {Meta, Source} from "@storybook/blocks";
+
+import TokenTable from "../../.storybook/components/token-table";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import * as tokens from "@khanacademy/wonder-blocks-tokens";
+
+import ComponentInfo from "../../.storybook/components/component-info";
+import packageConfig from "../../packages/wonder-blocks-tokens/package.json";
+
+<Meta title="Packages / Tokens / Media Query Breakpoints" />
+
+# Media Queries
+
+<ComponentInfo name={packageConfig.name} version={packageConfig.version} />
+
+All the available media query breakpoint values and pure widths that can be used for min-width,
+max-width, width, etc.
+
+## Usage
+
+### Media Queries
+
+You can use mediaQuery conditions by importing `breakpoint` from the
+`wonder-blocks-tokens` package and accessing its object properties for sizing like so:
+`breakpoint.mediaQuery.sm`.
+
+```js
+import {breakpoint} from "@khanacademy/wonder-blocks-tokens";
+const styles = {
+    [breakpoint.mediaQuery.sm]: {
+        flexDirection: "column"
+    }
+};
+```
+
+### Pure widths
+
+You can also use pure width values by importing `breakpoint` from the
+`wonder-blocks-tokens` package and accessing the `width` object:
+`breakpoint.width.sm`. These can be useful for tooling or in CSS where a number value
+is needed. Therefore, pixel values are returned without a unit. You can interpolate
+a string to add the `px` unit like so:
+
+```js
+import {breakpoint} from "@khanacademy/wonder-blocks-tokens";
+const styles = {
+    element: {
+        maxWidth: `${breakpoint.width.lg}px`
+    }
+};
+```
+
+## Tokens
+
+<TokenTable
+    columns={[
+        {
+            label: "Media Query Token",
+            cell: (row) => <code>breakpoint.mediaQuery.{row.label}</code>,
+        },
+        {
+            label: "Value",
+            cell: "value",
+        },
+    ]}
+    tokens={tokens.breakpoint.mediaQuery}
+/>
+
+<TokenTable
+    columns={[
+        {
+            label: "Width Token",
+            cell: (row) => <code>breakpoint.width.{row.label}</code>,
+        },
+        {
+            label: "Value",
```

Comment:
Since the values are numbers now instead of a string with "px", we can specify the unit in the column label! 

```suggestion
            label: "Value (px)",
```